### ISSUE TITLE
Snapchat keeps crashing when I am trying to open it, and I've tried everything.

### ISSUE THREAD

**Issue by MohammedAli0** [rhunk/SnapEnhance#1237]
### Description

Hello, so basically 2 days ago Snapchat just stopped working and immediately crashes whenever I try to open it. I was using Snapchat 12.81 or whatever version that SnapEnhance 2.1 was tested on, with SnapEnhance 2.1 release version. After it started crashing, I tried clearing the cache and that didn't work. Then, I tried deleted Snapchat and using lspatch to patch Snapchat again with snapenhance, the same apk, and same snapenhance app, but didn't work. Then, I tried an older version, like 11. something, with same snapenhance 2.1, didn't work and still crashes before even having a chance to load anything. Then, I tried the newest version, and again it didn't work. Atp, I gave up and used normal snap for a day. Then, I came back, I deleted everything, snapenhancr and Snapchat, all the apks and deleted the trash just to try with a fresh start. I downloaded the 12.81 apk again, and also downloaded snapenhance release version and patched it, still didn't work. THEN, I found someone with the same issue, and one of the developers recommended to use the debug version. I downloaded the debug version using nightly link, used a vpn just incase, patched 12.81 snap with it, and it still doesn't work, just instant crash. pls help!

### Reproduction steps

idk, happened randomly.

### Logs

```markdown
debug won't show anything related to the crashes.
```

### Snapchat Version

12.81.0.44

### SnapEnhance Version

2.1

### Agreement

- [X] This is not a bug regarding Snapchat+.
- [X] I have provided a detailed description of the issue.
- [ ] I have attached a log if deemed necessary.
- [ ] This issue is not a duplicate.

**Comment by DarkKnight17** [comment]
Maybe your acc is banned? Go to [Snapchat Site](https://accounts.snapchat.com/v2/welcome) then check if acc details

**Comment by MohammedAli0** [comment]
> Maybe your acc is banned? Go to [Snapchat Site](https://accounts.snapchat.com/v2/welcome) then check acc details

Nope, I am able to use snapchat without snapenhance for like half a day, and I have 2 accounts. But, just now, I tried to use snapchat and it says snapchat detected that I was using a 3rd party application, and it isn't my accounts because both of them are working and not banned, it's snapenhance. I think I have to delete snapchat and re-download it now everytime because snapchat detected that I am using a 3rd party app, even though I have snap enhance deleted rn.

**Comment by AdverseMiller** [comment]
It is not snapenhance. Snapchat will no longer open if it detects you're fucking with it

**Comment by MohammedAli0** [comment]
> It is not snapenhance. Snapchat will no longer open if it detects you're fucking with it

exactly? do you not know what SnapEnhance is 💀 snapenhance is the problem here, it got detected.

**Comment by AdverseMiller** [comment]
> exactly? do you not know what SnapEnhance is 💀 snapenhance is the problem here, it got detected.

Well yeah, no shit. Snap itself is willingly blocking itself from opening because of the tweak. So sure, you can indirectly say it's because of snapenhance, but the tweak itself isn't crashing the app lol

**Comment by MohammedAli0** [comment]
> > exactly? do you not know what SnapEnhance is 💀 snapenhance is the problem here, it got detected.
> 
> Well yeah, no shit. Snap itself is willingly blocking itself from opening because of the tweak. So sure, you can indirectly say it's because of snapenhance, but the tweak itself isn't crashing the app lol

I see where you're coming from, but how would I find what's causing the crash? I'm guessing lspatch or smthing like that.

**Comment by AdverseMiller** [comment]
Nothing about what you're doing it causing the crash. Snapchat *itself* is crashing.

**Comment by suryadip2008** [comment]
> > > exactly? do you not know what SnapEnhance is 💀 snapenhance is the problem here, it got detected.
> > 
> > 
> > Well yeah, no shit. Snap itself is willingly blocking itself from opening because of the tweak. So sure, you can indirectly say it's because of snapenhance, but the tweak itself isn't crashing the app lol
> 
> I see where you're coming from, but how would I find what's causing the crash? I'm guessing lspatch or smthing like that.

[Issue Disclaimer](https://t.me/snapenhance_chat/1/127787)

**Comment by MohammedAli0** [comment]
> Nothing about what you're doing it causing the crash. Snapchat _itself_ is crashing.

wdym? snapchat won't crash if it's not patched with snapenhance.
